<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetSearch</class>
 <widget class="QWidget" name="WidgetSearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>189</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitterSearch">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QFrame" name="frameSearchSidebar">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_20">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QScrollArea" name="scrollAreaSearchSidebar">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QFrame" name="frameSearchSidebarContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>184</width>
            <height>522</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_19">
           <property name="leftMargin">
            <number>4</number>
           </property>
           <property name="topMargin">
            <number>4</number>
           </property>
           <property name="rightMargin">
            <number>4</number>
           </property>
           <property name="bottomMargin">
            <number>4</number>
           </property>
           <property name="spacing">
            <number>4</number>
           </property>
           <item row="0" column="0">
            <widget class="QToolButton" name="toolButtonNewSearch">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>New Search</string>
             </property>
             <property name="icon">
              <iconset resource="../Resource.qrc">
               <normaloff>:/Resource/Generic/NewSearch.png</normaloff>:/Resource/Generic/NewSearch.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextBesideIcon</enum>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <layout class="QGridLayout" name="gridLayoutSearchResultsTask">
             <property name="spacing">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QToolButton" name="toolButtonSearchResultsTaskHeader">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Results</string>
               </property>
               <property name="icon">
                <iconset resource="../Resource.qrc">
                 <normaloff>:/Resource/Network/BrowseStatistics.png</normaloff>:/Resource/Network/BrowseStatistics.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextBesideIcon</enum>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QFrame" name="frameSearchResultsTask">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Sunken</enum>
               </property>
               <layout class="QGridLayout" name="gridLayout_27">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <property name="horizontalSpacing">
                 <number>0</number>
                </property>
                <property name="verticalSpacing">
                 <number>2</number>
                </property>
                <item row="1" column="0">
                 <widget class="QLabel" name="labelSearchResultsSearchingHeader">
                  <property name="font">
                   <font>
                    <pointsize>10</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Searching:</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="labelSearchResultsFoundHeader">
                  <property name="font">
                   <font>
                    <pointsize>10</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Found:</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="labelSearchResultsSearching">
                  <property name="text">
                   <string>Not Currently Searching</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="labelSearchResultsFound">
                  <property name="text">
                   <string>No Files Found</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
           <item row="6" column="0">
            <layout class="QGridLayout" name="gridLayoutSearchFiletypeTask">
             <property name="spacing">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QToolButton" name="toolButtonSearchFiletypeTaskHeader">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Any File Type</string>
               </property>
               <property name="icon">
                <iconset resource="../Resource.qrc">
                 <normaloff>:/Resource/Generic/Search.png</normaloff>:/Resource/Generic/Search.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextBesideIcon</enum>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QFrame" name="frameSearchFiletypeTask">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Sunken</enum>
               </property>
               <layout class="QGridLayout" name="gridLayout_25">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <property name="horizontalSpacing">
                 <number>0</number>
                </property>
                <property name="verticalSpacing">
                 <number>2</number>
                </property>
                <item row="4" column="0">
                 <layout class="QGridLayout" name="gridLayoutSearchSize">
                  <item row="0" column="0">
                   <widget class="QComboBox" name="comboBoxSearchSizeMin">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLabel" name="labelSearchSizeTo">
                    <property name="text">
                     <string>to</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="2">
                   <widget class="QComboBox" name="comboBoxSearchSizeMax">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="labelSearchSize">
                  <property name="text">
                   <string>File size must be:</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="0">
                 <widget class="QListView" name="listViewSearchFiletype">
                  <property name="alternatingRowColors">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
           <item row="4" column="0">
            <layout class="QGridLayout" name="gridLayoutSearchNetworksTask">
             <property name="spacing">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QToolButton" name="toolButtonSearchNetworksTaskHeader">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Networks</string>
               </property>
               <property name="icon">
                <iconset resource="../Resource.qrc">
                 <normaloff>:/Resource/Network/Network.png</normaloff>:/Resource/Network/Network.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextBesideIcon</enum>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QFrame" name="frameSearchNetworksTask">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Sunken</enum>
               </property>
               <layout class="QGridLayout" name="gridLayout_23">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <property name="horizontalSpacing">
                 <number>0</number>
                </property>
                <property name="verticalSpacing">
                 <number>2</number>
                </property>
                <item row="1" column="0">
                 <widget class="QLabel" name="labelSearchNetworks">
                  <property name="text">
                   <string>Search on these networks:</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <layout class="QGridLayout" name="gridLayoutSearchNetworks">
                  <item row="0" column="0">
                   <widget class="QCheckBox" name="checkBoxSearchGnutella2">
                    <property name="text">
                     <string>Gnutella 2</string>
                    </property>
                    <property name="icon">
                     <iconset resource="../Resource.qrc">
                      <normaloff>:/Resource/Networks/Gnutella2.png</normaloff>:/Resource/Networks/Gnutella2.png</iconset>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QCheckBox" name="checkBoxSearchAres">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="text">
                     <string>Ares</string>
                    </property>
                    <property name="icon">
                     <iconset resource="../Resource.qrc">
                      <normaloff>:/Resource/Networks/Ares.png</normaloff>:/Resource/Networks/Ares.png</iconset>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QCheckBox" name="checkBoxSearchEDonkey">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="text">
                     <string>eDonkey</string>
                    </property>
                    <property name="icon">
                     <iconset resource="../Resource.qrc">
                      <normaloff>:/Resource/Networks/EDonkey.png</normaloff>:/Resource/Networks/EDonkey.png</iconset>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QGridLayout" name="gridLayoutSearchTask">
             <property name="spacing">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QToolButton" name="toolButtonSearchTaskHeader">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Search</string>
               </property>
               <property name="icon">
                <iconset resource="../Resource.qrc">
                 <normaloff>:/Resource/Generic/Search.png</normaloff>:/Resource/Generic/Search.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextBesideIcon</enum>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QFrame" name="frameSearchTask">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>160</height>
                </size>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Sunken</enum>
               </property>
               <layout class="QGridLayout" name="gridLayout_21">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <property name="horizontalSpacing">
                 <number>0</number>
                </property>
                <property name="verticalSpacing">
                 <number>2</number>
                </property>
                <item row="1" column="0" colspan="2">
                 <widget class="QLabel" name="labelSearch">
                  <property name="text">
                   <string>Type your search here:</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0" colspan="2">
                 <widget class="QLineEdit" name="lineEditSearch"/>
                </item>
                <item row="3" column="0" colspan="2">
                 <widget class="QLabel" name="labelSearchFileType">
                  <property name="text">
                   <string>File Type:</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0" colspan="2">
                 <widget class="QComboBox" name="comboBoxSearchFileType">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                 </widget>
                </item>
                <item row="5" column="0" colspan="2">
                 <layout class="QGridLayout" name="gridLayoutSearchTaskButtons">
                  <item row="0" column="0">
                   <widget class="QToolButton" name="toolButtonSearch">
                    <property name="text">
                     <string>Search</string>
                    </property>
                    <property name="icon">
                     <iconset resource="../Resource.qrc">
                      <normaloff>:/Resource/Generic/Search.png</normaloff>:/Resource/Generic/Search.png</iconset>
                    </property>
                    <property name="toolButtonStyle">
                     <enum>Qt::ToolButtonTextBesideIcon</enum>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QToolButton" name="toolButtonSearchClear">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="text">
                     <string>Clear</string>
                    </property>
                    <property name="icon">
                     <iconset resource="../Resource.qrc">
                      <normaloff>:/Resource/Generic/Cancel.png</normaloff>:/Resource/Generic/Cancel.png</iconset>
                    </property>
                    <property name="toolButtonStyle">
                     <enum>Qt::ToolButtonTextBesideIcon</enum>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
           <item row="10" column="0">
            <spacer name="verticalSpacerSearchSidebar">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frameSearchResults">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayoutSearchResults">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>toolButtonSearchTaskHeader</sender>
   <signal>toggled(bool)</signal>
   <receiver>frameSearchTask</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>60</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonSearchNetworksTaskHeader</sender>
   <signal>toggled(bool)</signal>
   <receiver>frameSearchNetworksTask</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonSearchFiletypeTaskHeader</sender>
   <signal>toggled(bool)</signal>
   <receiver>frameSearchFiletypeTask</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonSearchResultsTaskHeader</sender>
   <signal>toggled(bool)</signal>
   <receiver>frameSearchResultsTask</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>397</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>420</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditSearch</sender>
   <signal>returnPressed()</signal>
   <receiver>toolButtonSearch</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
